pywinauto/unittests/test_hwndwrapper.py

```diff
@@ -246,6 +251,10 @@ def testTopLevelParent(self):
         self.assertEqual(self.ctrl.top_level_parent(), self.dlg.handle)
         self.assertEqual(self.dlg.top_level_parent(), self.dlg.handle)
 
+    def test_issue_760_desktop_win32(self):
```

Comment:
Issue number should be in test name only if it is a bug. Name it `test_get_active_desktop` by feature name. The same for base wrapper.